Locker/Locker/Locker.swift

```diff
@@ -182,8 +170,8 @@ public extension Locker {
     }
 }
 
-private extension Locker {
-    private static func setSecretForDevice(_ secret: String, for uniqueIdentifier: String, completion: ((LockerError?) -> Void)? = nil) {
+public extension Locker {
```

Comment:
Can you try with `internal`? 🤔 

Marking it as `public` will expose this method to the user. A better approach would be keeping this contained in the lib.